<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据管理系统</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>701</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="manageBtn">
      <property name="text">
       <string>商品管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="chartBtn">
      <property name="text">
       <string>销售统计</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="passwdBtn">
      <property name="text">
       <string>修改密码</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>70</y>
     <width>700</width>
     <height>410</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="managePage">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>501</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>商品管理</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>351</width>
       <height>331</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>347</width>
        <height>237</height>
       </rect>
      </property>
      <attribute name="label">
       <string>出售商品</string>
      </attribute>
      <widget class="QSpinBox" name="sellNumSpinBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>120</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="sellCancelBtn">
       <property name="geometry">
        <rect>
         <x>167</x>
         <y>210</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>10</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>类型：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>47</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>品牌：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>37</x>
         <y>160</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>金额：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>80</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>单价：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>120</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>数量：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sellSumLineEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="sellOkBtn">
       <property name="geometry">
        <rect>
         <x>47</x>
         <y>210</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>197</x>
         <y>80</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>元</string>
       </property>
      </widget>
      <widget class="QComboBox" name="sellBrandComboBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>47</y>
         <width>210</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="sellPriceLineEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>196</x>
         <y>160</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>元</string>
       </property>
      </widget>
      <widget class="QComboBox" name="sellTypeComboBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="sellLastNumLabel">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>120</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>剩余数量：000</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>347</width>
        <height>237</height>
       </rect>
      </property>
      <attribute name="label">
       <string>已有商品入库</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>347</width>
        <height>237</height>
       </rect>
      </property>
      <attribute name="label">
       <string>新商品入库</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QListWidget" name="daliyList">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>41</y>
       <width>331</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
